src/components/Navbar.vue

```diff
@@ -1,27 +1,32 @@
 <template>
-    <nav class="navbar">
+    <nav v-if="showNavbar" class="navbar">
+       
         <div class="logo">
             <img :src="keyclashlogo" alt="KeyClash Logo">
         </div>
         <ul class="nav-links">
             <li><a href="/">Home</a></li>
-            <li><a href="/#about">About</a></li>
-            <li><a href="/#about-us">About us</a></li>
+            <li><a href="/#about-game">About The Game</a></li>
+            <li><a href="/#meet-team">Meet The Team</a></li>
+            <li><router-link :to="{ path: '/Userauth', query: { mode: 'register' }}" class="registration">Register</router-link></li>
```

Comment:
Is router-link an element defined by you, or is this built into Vue or some other service? how does this work?